README.md

```diff
@@ -35,6 +35,11 @@ Activate the module by adding it to the config.js file as shown in the example b
 | items            | The rows of data you want the module to show. See list below. By default will show all available            | `['battery','range-estimated','locked','odometer']` |
 | initialLoadDelay | How many seconds to delay initial API call                                                                  |
 | dataTimeout      | How old data must be in seconds before 'data-time' is displayed. Use 0 to always show                       | `0`                                                 |
+| googleMapApiKey  | Google Maps "Static Maps API" | `AIzaSyB6KgHKwRNa63JsVHuu7d8jV-1IH875idKs` |
+| mapZoom          | (optional) Zoom Level of map                                                                                | 13                                                  |
+| mapWidth         | (optional) Specify width of map                                                                             | 300                                                 |
+| mapHeight        | (optional) Specify height of map                                                                            | 150                                                 |
+| excludeLocations | (optional) Specify locations to hide map by TeslaFi Location Tags (case insensitive) - stops Google Map generation @ listed locations | `[ 'home', 'Work', 'cottage' ]` |
```

Comment:
I think the wording could use some work here.

Perhaps "Specify TeslaFi tagged locations at which the `map` field will not be shown. Case insensitive. " or something like that?